src/wp-admin/css/media.css

```diff
@@ -192,13 +186,13 @@
 
 .media-item .original {
 	position: relative;
-	min-height: 34px;
+	margin: 7px 6px;
```

Comment:
I think this top margin adds a 'gap' between the rows while the upload is in progress. Screenshot:
![Screenshot 2024-05-22 at 12 01 17](https://github.com/WordPress/wordpress-develop/assets/1682452/98942173-cee3-45bb-92a2-01ce3619a916)

The gap then disappears when the upload is complete. Ideally, there should be no gap.

Reply:
I have added those css for grid layout but forgot to remove it. Just updated CSS.